docs/config/_default/hugo.toml

```diff
@@ -61,10 +61,83 @@ weight = 1
 
 [[menu.main]]
     name = "GitHub"
-    weight = 50
+    weight = 60
+    url = "https://github.com/cilium/tetragon/"
+    pre = "<i class=\"fa-brands fa-github\"></i>"
+
+[[menu.navbar_main]]
+    name = "Documentation"
+    weight = 10
+    url = "/docs"
+
+[[menu.navbar_main]]
+    name = "Community"
+    weight = 20
+    url = "https://cilium.io/get-involved/"
+
+[[menu.navbar_main]]
+    name = "Getting Started"
+    weight = 30
+    url = "/docs/getting-started"
+
+[[menu.navbar_main]]
+    name = "Enterprise"
+    weight = 40
+    url = "/enterprise"
+
+[[menu.navbar_main]]
+    name = "GitHub"
+    weight = 60
     url = "https://github.com/cilium/tetragon/"
     pre = "<i class=\"fa-brands fa-github\"></i>"
 
+[[menu.navbar_main]]
+    name = "Use Cases"
+    weight = 50
+    identifier = "use-cases"
+
+[[menu.navbar_main]]
+    name = "Capabilities Monitoring"
+    parent = "use-cases"
+    url = "/use-cases/capabilities-monitoring"
+    weight = 1
```

Comment:
It looks like your version looks great and I have not dive into the technical details, so maybe ignore this totally: but I was wondering if it would have been possible to do something like `[[menu.navbar_main.use-cases]]` and then declare each use-cases using this.